dockerfiles/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: "3"
```

```suggestion
version: "3.7"
```
This looks required for the `init` parameter for me